configuration/product_variant/coupon.csv

```diff
@@ -0,0 +1,3 @@
+id,lst_price,name,categ_id/id,type
+__export__.product_product_157,-50,Coupon,product.coupon,service
+__export__.product_product_300,-60,Coupon,product.coupon,service
```

Comment:
Why are there 2 lines here?
Is this intentional?

Reply:
Removed...It second line was added accidentally